portafly/src/components/pages/accounts/listing/AccountsListingPage.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { useAsync } from 'react-async'
+import { useDocumentTitle, Loading, DataListProvider } from 'components'
+import { AccountsListingTable, generateColumns, generateRows } from 'components/pages/accounts/listing'
+import {
+  Alert,
+  PageSection,
+  TextContent,
+  Text,
+  Button,
+  Flex,
+  FlexItem,
+  FlexModifiers
+} from '@patternfly/react-core'
+import { getDeveloperAccounts } from 'dal/accounts'
+import { useTranslation } from 'i18n/useTranslation'
+import { PlusCircleIcon, ExportIcon } from '@patternfly/react-icons'
+
+const AccountsListingPage: React.FunctionComponent = () => {
+  const { data: accounts, error, isPending } = useAsync(getDeveloperAccounts)
+  const { t } = useTranslation('audienceAccountsListing')
+  useDocumentTitle(t('title_page'))
+
+  const isMultitenant = false // TODO: get this somehow
+
+  const DataListTable = ({ accounts: _accounts }: any) => {
```

Comment:
`DataListTable` can be moved outside `AccountsListingPage`

Reply:
Well, DataListTable is simply a wrap around the Provider and a way to render it only when `accounts !== undefined`. You could see it as a `const` but instead if benefits from React's nomenclature. It's just a way to keep [line 75](https://github.com/3scale/porta/pull/1820/files#diff-db85736b3e028a1b4f2e1a41af362af2R72) clean:
```jsx
{accounts && <DataListTable accounts={accounts} />}
```

Reply:
Having it outside will benefit of a better garbage collection. If `accounts` is falsey, then `DataListTable` will be removed completely (when outside). Either way, it stays in the closure

Reply:
OK